src/platform/forms/tests/migrations.unit.spec.js

```diff
@@ -34,6 +36,9 @@ const mappedIds = [
 
 const configs = [
   fullSchema1010ez,
+  // Remap the formId to match the name in vets-json-schema
+  // This should only affect the mapping in the "check all forms" test
+  { ...fullSchema526AllClaims, formId: '21-526EZ-ALLCLAIMS' },
```

Comment:
dumb question, but I'm guessing the reason we can't simply change the `formId` in the actual form config is for backwards compatibility w/ 526 v1?

Reply:
Kind of? The `formId` affects SiP, so if we change it, it'll require some back end changes, too.

Honestly, it was a mistake to keep the `formId` the same for v2 as it was for v1. We didn't realize what kind of mistake it would be, but it's...a pain. The original intention was to piggyback on the v1 work on the back end, but... :grimacing: Hindsight is 20/20.

At this point, it probably doesn't make sense to change it, as much as I'd like to. :disappointed: